src/app/presentation/progress-bar/progress-bar.component.html

```diff
@@ -1,3 +1,6 @@
 <div class="progress-bar">
-  <div class="progress" [style.width.%]="progress"></div>
+  <div class="points-container" (mouseenter)="isHovered = true" (mouseleave)="isHovered = false">
+    <div class="slide-block" *ngFor="let slide of slides" [class.expanded]="isHovered" [class.exercise-block]="slide.isExercise" [class.completed-slide]="slide.slideId <= currentSlideId" (click)="goToSlide(slide.slideId)">
```

Comment:
Can you achieve the same result with css `:hover` selector? It would probably be more performant to use css for this unless you need to do something complex with this event.